pkg/lib/cockpit-components-select.jsx

```diff
@@ -118,11 +119,15 @@ export class StatelessSelect extends React.Component {
         if (this.props.enabled === false)
             buttonClasses += " disabled";
 
+        let iconClass = "caret";
+        if (this.props.iconClass)
+            iconClass = this.props.iconClass;
```

Comment:
Please shorten that to `const iconClass = this.props.iconClass || "caret";`

Also, please adjust the commit message to "lib: Enable setting custom icon in Select component"